tests/Smoke/InstallViaComposerTest.php

```diff
@@ -72,34 +91,18 @@ public static function setUpBeforeClass(): void
         }
     }
 
-    public function testInstallationViaPathIsPossible(): void
+    protected function setUp(): void
     {
-        $fs = new Filesystem();
-
-        $tmpPath = tempnam(sys_get_temp_dir(), 'cs_fixer_tmp_');
-        unlink($tmpPath);
-        $fs->mkdir($tmpPath);
-
-        $initialComposerFileState = [
-            'repositories' => [
-                [
-                    'type' => 'path',
-                    'url' => __DIR__.'/../..',
-                ],
-            ],
-            'require' => [
-                'friendsofphp/php-cs-fixer' => '*@dev',
-            ],
-        ];
+        $this->fs = new Filesystem();
```

Comment:
should we create new `fs` instance for each `setup` call?
or maybe move to constructor?